Makefile

```diff
@@ -27,6 +28,10 @@ go-build:
 	@echo "  >  Building binaries..."
 	@CGO_ENABLED=0 go build -o ${PROJECTNAME} .
```

Comment:
What if we run a build on arm platform? Then `go-build` would produce a binary for arm as well as `go-compile-arm` and no way to compile for `amd64`

Would it make sense to pass a variable into make instead and reuse `go-build`
```make
GOARCH ?= $(shell go env GOARCH) # detect automatically the underlying arch
...
go-build:
    @CGO_ENABLED=0 GOARCH=$(GOARCH) go build -o ${PROJECTNAME} .
```

if we need to cross-compile

```bash
GOARCH=arm64 make
# or
GOARCH=amd64 make
```